diff --git a/node_modules/axios/lib/adapters/adapters.js b/node_modules/axios/lib/adapters/adapters.js
index 5d18745..debcd3f 100644
--- a/node_modules/axios/lib/adapters/adapters.js
+++ b/node_modules/axios/lib/adapters/adapters.js
@@ -1,7 +1,6 @@
 import utils from '../utils.js';
 import httpAdapter from './http.js';
 import xhrAdapter from './xhr.js';
-import * as fetchAdapter from './fetch.js';
 import AxiosError from '../core/AxiosError.js';
 
 /**
@@ -16,9 +15,6 @@ import AxiosError from '../core/AxiosError.js';
 const knownAdapters = {
   http: httpAdapter,
   xhr: xhrAdapter,
-  fetch: {
-    get: fetchAdapter.getFetch,
-  },
 };
 
 // Assign adapter names for easier debugging and identification
diff --git a/node_modules/axios/lib/defaults/index.js b/node_modules/axios/lib/defaults/index.js
index b83c2c0..cee2636 100644
--- a/node_modules/axios/lib/defaults/index.js
+++ b/node_modules/axios/lib/defaults/index.js
@@ -36,7 +36,7 @@ function stringifySafely(rawValue, parser, encoder) {
 const defaults = {
   transitional: transitionalDefaults,
 
-  adapter: ['xhr', 'http', 'fetch'],
+  adapter: ['xhr', 'http'],
 
   transformRequest: [
     function transformRequest(data, headers) {
